### DISCUSSION TITLE
3 day old Pending blocks not processed

### DISCUSSION THREAD

**Discussion by ei8ht187** [coinfoundry/miningcore#1439] [Q&A] [answered]
Hello, 
title says it all. There was a problem that lead to over 4000 pending blocks. Is there a way to tell MC to rescan this from the wallet? 
They are not orphaned or so.

BR

ei8ht

**Comment by oliverw** [comment]
MC lost track of mined blocks?

**Comment by mboyd1** [comment, marked as answer]
I've had this happen, miningcore submits block, daemon accepts block (miningcore logs Daemon accepted block), the block is on-chain and multiple confirmations but when payout handler runs it gets an ambiguous error -5 and marks the block as orphaned incorrectly
 Block [height] classified as orphaned due to daemon error -5

not sure if I can manually intervene by updating the blocks table in the database to change status to confirmed and add the block reward so payouts will work?

**Reply by oliverw** [reply]
> I've had this happen, miningcore submits block, daemon accepts block (miningcore logs Daemon accepted block), the block is on-chain and multiple confirmations but when payout handler runs it gets an ambiguous error -5 and marks the block as orphaned incorrectly Block [height] classified as orphaned due to daemon error -5
> 
> not sure if I can manually intervene by updating the blocks table in the database to change status to confirmed and add the block reward so payouts will work?

That's the recommended solution.

**Reply by mboyd1** [reply]
> That's the recommended solution.

thanks, turns out I had it talking to the wrong daemon that lacked the privkey thus the error I was receiving.  once I switched it back to talk to correct daemon and set block to 'pending' status it picked it up as normal
